Documentation/getting-started.md

```diff
@@ -1,6 +1,8 @@
 # CoreOS &#43; Kubernetes Step By Step
 
-This guide will walk you through a deployment of a single-master/multi-worker Kubernetes cluster on CoreOS. We're going to:
+This guide will walks through the deployment of a single-master/multi-worker Kubernetes cluster on CoreOS. This is a long form "hard way" guide that can be error prone but can help you understand how the system works. If you want to get started faster check out [Tectonic Free Tier][tectonic-free] and the [Tectonic Installer][tectonic-installer].
```

Comment:
first sentence: "This guide walks through deploying a Kubernetes cluster of CoreOS nodes, with a single controller and multiple workers."
second sentence: "This guide enumerates the multiple steps and stages of a Kubernetes deployment."
third sentence: "To quickly deploy a Kubernetes cluster without engaging component-level details, check out the [free tier of the CoreOS Tectonic][tectonic-free] Kubernetes distribution, or the [open-source Tectonic Installer][tectonic-installer] that drives Tectonic's automation of cluster deployments."